installation/template/js/remove.js

```diff
@@ -36,37 +36,68 @@ if (document.getElementById('removeInstallationFolder')) {
 			e.preventDefault();
 			let confirm = window.confirm(Joomla.Text._('INSTL_REMOVE_INST_FOLDER').replace('%s', 'installation'));
 			if (confirm) {
-				Joomla.request({
-					method: "POST",
-					url: Joomla.installationBaseUrl + '?task=installation.removeFolder&format=json',
-					perform: true,
-					token: true,
-					headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
-					onSuccess: function (response) {
+			    Joomla.deleteJoomlaInstallationDirectory();
+			}
+		});
+}
+
+const completeInstallationOptions = document.querySelectorAll('.complete-installation');
+
+completeInstallationOptions.forEach(function(item) {
+    item.addEventListener('click', function (e) {
+        // Evil voodoo. Once a button is clicked ensure they don't spam click it...
+        completeInstallationOptions.forEach(function(nestedItem) {
+            nestedItem.disabled = true;
+        });
+
+        // In development mode we show the user a pretty button to allow them to choose whether to delete the installation
+        // directory or not. In stable the decision is made forward. Maximum extermination!
```

Comment:
```suggestion
        // directory or not. In a stable release their is no choice. The installation directory is always removed.
```

I think you need some sleep ;)

Reply:
s/their/there

Reply:
Can I not have one doctor who reference :P But also yes probably 1.5 hours isn't enough :)

Reply:
also isn't it "there" in this case?

Reply:
"Maximum extermination!" ... daleks around? ;-)

Reply:
> also isn't it "there" in this case?

There is in Brians suggestion... which is the wrong use of "their"

Reply:
yep:

> In a stable release their is no choice. The installation directory is always removed.

should be "there".

But why not just use:

> In a stable release the installation directory is always removed.